GANDLF/optimizers/README.md

```diff
@@ -3,10 +3,12 @@
 ## Adding a new algorithm
 
 - For an optimizer defined in PyTorch [[ref](https://pytorch.org/docs/stable/optim.html#algorithms)], update the `GANDLF.optimizers.wrap_torch.py` submodule.
-- For a custom optimizer, create a new submodule called `GANDLF.optimizers.${awesome_optimizer}.py`. Ensure that it inherits from PyTorch's base optimizer class [[ref](https://pytorch.org/docs/stable/optim.html#base-class)]
+- For a custom optimizer, create a new submodule called `GANDLF.optimizers.${awesome_optimizer}.py`.
+- For a third-party optimizer (where the code is available from an external repository), add the relevant code under the `GANDLF.optimizers.thirdparty` submodule.
```

Comment:
```suggestion
- For a third-party optimizer (i.e., where the code is available from an external source/repository):
  - Add the relevant code under the `GANDLF.optimizers.thirdparty` submodule. 
  - Add a wrapper which takes in GaNDLF's `parameter` dictionary as input and creates a `torch.optim.Optimizer` object as output.
  - Add the wrapper to the `GANDLF.optimizers.thirdparty.__init__.py` so that it can be called from `GANDLF.optimizers.__init__.py`.
  - See `GANDLF.optimizers.thirdparty.adopy.py` as an example.
```